### PR TITLE
Fixes Always show full URLs sentence casing

### PR DESCRIPTION
**Pull request by Tonev**
Resolves

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: app/generated_resources.grd [hunk 1]
```diff
@@ -5937,7 +5937,7 @@ Keep your key file in a safe place. You will need it to create new versions of y
       </message>
       <if expr="use_titlecase">
```
Comment (emerick) [inline app/generated_resources.grd:5938]: I don't think this is quite the right fix, as this string is meant to be used when `use_titlecase` is in effect (for instance, I think in Mac menu items). Instead, I think this string should remain as-is and we'll need to break out the settings item into its own string that doesn't use title case at all.

**Review by emerick — CHANGES_REQUESTED** [review]

File: app/generated_resources.grd (hunk 1 above)
Comment (Tonev) [reply app/generated_resources.grd:5938]: Yes, I wasn't sure about it either. Thought I could give it a try after discussing the matter with Stephen (https://github.com/brave/brave-browser/issues/16939#issuecomment-1144058368), but it's all right if you decide to close this one as I'm not aware how to resolve this issue if the change I proposed is wrong. I'm really interested to see how it's fixed properly 👍

Comment (emerick) [reply app/generated_resources.grd:5938]: No worries! Your proposed fix was my first thought too; I just happened to notice the `use_titlecase` directive above it.

**Comment by Tonev** [comment]
I'll be closing this pull request as proposed changes aren't enough to resolve the issue.

**Pull request closed without merging**